.vscode/settings.json

```diff
@@ -15,6 +15,5 @@
   "mochaExplorer.require": "ts-node/register",
   "mochaExplorer.timeout": 20000,
   "prettier.disableLanguages": ["javascript", "typescript"],
-  "prettier.enable": true,
-  "tslint.enable": false
```

Comment:
is tslint obsolete? otherwise it may lead to weird behaviour when NOT disabling it

Reply:
tslint is not defined in the `extensions.json` AND:

> TSLint been deprecated in favor of ESLint - [Doc see first line in url](https://marketplace.visualstudio.com/items?itemName=ms-vscode.vscode-typescript-tslint-plugin)